src/metadata.rs

```diff
@@ -146,12 +146,18 @@ fn read_flac(path: &Path) -> Result<SongTags> {
 		.get("DISCNUMBER")
 		.and_then(|d| d[0].parse::<u32>().ok());
 	let year = vorbis.get("DATE").and_then(|d| d[0].parse::<i32>().ok());
+	let streaminfo = tag.get_blocks(metaflac::BlockType::StreamInfo);
```

Comment:
Nitpick, I think this variable should be called stream_info for consistency.

Reply:
streaminfo is the name of the block: https://xiph.org/flac/format.html#def_STREAMINFO
but if you want i can still rename to stream_info

Reply:
You're right. I was fooled by the name used by metaflac (StreamInfo).